src/main/java/com/jrjackson/RubyAnySerializer.java

```diff
@@ -237,11 +237,11 @@ private void serializeTime(RubyTime dt, JsonGenerator jgen, SerializerProvider p
         } else if (df instanceof RubyDateFormat) {
             // why another branch? I thought there was an easy win on to_s
             // maybe with jruby 9000
-            RubyDateFormat rdf = (RubyDateFormat) df.clone();
-            jgen.writeString(rdf.format(dt.getJavaDate()));
+            RubyDateFormat clonedRubyDatFormat = (RubyDateFormat) df.clone();
+            jgen.writeString(clonedRubyDatFormat.format(dt.getJavaDate()));
         } else {
-            SimpleDateFormat sdf = (SimpleDateFormat) df.clone();
-            jgen.writeString(df.format(dt.getJavaDate()));
+            SimpleDateFormat clonedSimpleDateFormat = (SimpleDateFormat) df.clone();
```

Comment:
so the whole issue here was that we were using the wrong object?

Reply:
yup

Reply:
Could/Should we share a DateTimeFormatter instead of creating a new one here for every parse?

Reply:
I will take a look